app/router.js

```diff
@@ -12,28 +12,56 @@ import {
   removeLabel,
   reopenIssue,
   requestReviewers,
+  reportError,
   transferIssue,
 } from "./github.js";
 import { getAuthToken } from "./auth.js";
 import { extractCommaSeparated, extractUsersAndTeams } from "./converters.js";
+import { labelEnabled, transferEnabled } from "./command-enabled.js";
+import { defaultConfig } from "./default-config.js";
+
+const { Octokit } = require("@octokit/core");
```

Comment:
eslint should help you convert them to imports.

Reply:
👍 , yeah that format was just copied from docs. will fix